tests/lib/test_Repo.py

```diff
@@ -10,12 +10,12 @@
 #
 # instantiation
 #
-variants = {
-    'str': 'the-repo',
-    'Path': Path('the-repo'),
-}
-@pytest.mark.parametrize('variant', variants.values(), ids=variants.keys())
-def test_Repo_instantiate_types(tmp_path, variant):
+@pytest.mark.parametrize('variant', ['the-repo', Path('the-repo')])
```

Comment:
Retaining `ids` helps the readability of the output of `pytest` when tests are failing. It doesn't know how to convert `Path('the-repo')` to a string.